.github/workflows/pull_requests.yaml

```diff
@@ -5,7 +5,10 @@ on: [pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest', 'macos-latest']
+    runs-on: ${{ matrix.os }}
```

Comment:
I love that we are dipping our toes in here, but I would caution that we don't spend too much time on cross-compatibility until we're past v1, or we risk spending a lot of time in maintenance of platforms not in active use.

Reply:
Although I would say that OSX and Linux are must-have...